### DISCUSSION TITLE
Starting charging session

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [leeyuentuen/alfen_wallbox#93] [Q&A]
Hi!

Thanks for creating this integration, it looks great! The thing is, I can't seem to understand how to start a charging session? I currently try to set the charging amps to 16. The HA UI shows a green success button when I do that, but nothing happens.

What steps am I missing?

**Comment by leeyuentuen** [comment]
if you use green, it will check the percentage. If you don't have enough power (e.g. solar panel power) the charging will not start. unless you set your green share low enough.
100% mean that all the power should be from solar panel, 0% means it will use all the power from the net

**Comment by keesvanbemmel** [comment]
Not exactly what I was looking for. If the car is connected but the Alfen eve is unlocked I don't know which service(s) to call from this integration to actually start charging.

**Reply by leeyuentuen** [reply]
> Not exactly what I was looking for. If the car is connected but the Alfen eve is unlocked I don't know which service(s) to call from this integration to actually start charging.

if it is unlocked the  it start charging, but you can set the charging amp to 0, then it will not 'charge' but still unlocked.
